src/code/z_camera.c

```diff
@@ -4343,9 +4360,9 @@ s32 Camera_Subj4(Camera* camera) {
     OLib_Vec3fDiffToVecSphGeo(&sp5C, at, eye);
     sCameraInterfaceFlags = roData->interfaceFlags;
     if (camera->animState == 0) {
-        spA4 = Camera_GetCamBgDataUnderPlayer(camera, &spAA);
-        Camera_Vec3sToVec3f(&rwData->unk_00.point, &spA4[1]);
-        Camera_Vec3sToVec3f(&sp98, &spA4[spAA - 2]);
+        bgCamData = (Vec3s*)Camera_GetBgCamDataUnderPlayer(camera, &bgCamDataCount);
+        Camera_Vec3sToVec3f(&rwData->unk_00.point, &bgCamData[1]);
+        Camera_Vec3sToVec3f(&sp98, &bgCamData[bgCamDataCount - 2]);
```

Comment:
Maybe variables `bgCamData` and `bgCamDataCount` could be named around crawlspace points to avoid confusion with the `BgCamData` struct and make things a bit clearer, unless you already plan to do this in a dedicated crawlspace documentation pass?

Reply:
Good call, I use this `bgCamData` in my crawlspace docs as well but it can change. If it's a list of positions along the crawlspace line, perhaps `crawlspaceLinePosList`? I feel like that could be abbreviated more

Reply:
I think I would just go with `crawlspacePosList` or `crawlspacePoints`. I see the "line" part mostly as a limitation with the current implementation, but in theory the data could be anything, including turns or height.
But either way long names aren't as bad for local variables like these, so even what you suggested could be fine